Changelog.md

```diff
@@ -12,6 +12,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 * [ALFREDOPS-687] Add image for 6.2.2.
 * [DOCKER-275] Introduce variables for direct communication share-alfresco. Make sure that ALFRESCO_INTERNAL_HOST is set to the correct name of the alfresco service/container.
 * [DEVEM-478] Share 7.0.0
+* [BNPPFALF7-6] Share 7.0.1.2
```

Comment:
Please don't reference customer jira tickets in public repositories

Reply:
To rename branch too, I will start a new PR #21 then and close this one.